python/tox.ini

```diff
@@ -11,9 +11,7 @@ envlist =
     build-dist
     ; Mirror Supported LTS DBR versions here: https://docs.databricks.com/release-notes/runtime/
     ; Use correct PySpark version based on Python version present in env name
-    py37-pyspark300,
-    py38-pyspark{312,321},
-    py39-pyspark{330,332}
+    dbr{91,104,113,122,133,142}
```

Comment:
My opinion, but I'm opposed to using DBR in our env names for tox since we want to make this friendly to the open-source community as well.

Reply:
Ya I get what you're saying. Any recommendations for a better naming convention? @tnixon what are your thoughts?

Reply:
I understand your concern, @R7L208 - but we're not actually creating a dependency on databricks or these runtime versions. The larger goal is just to test across a range of environment configurations (versions of python, versions of spark, other dependencies). We choose the LTS DBR versions as a known quantity for this (and, of course, because Databricks customers will be running on one of these). 

I guess we could come up with a more "neutral" naming convention, but the truth is the configurations come from the DBR versions, and I like that the names tell us quickly which one to reference. So I would vote to stick with them. However, we should update the developer docs to explain the rationale for these names.

Reply:
Ya I think it's fine so as long as we make it abundantly clear in the docs that Databricks is not required for development.